pageserver/src/layered_repository.rs

```diff
@@ -240,7 +257,7 @@ impl LayeredRepository {
                 )?;
 
                 // List the layers on disk, and load them into the layer map
-                timeline.load_layer_map()?;
+                let _latest_timeline_on_disk = timeline.load_layer_map()?;
```

Comment:
This then gets sent to the upload queue in https://github.com/zenithdb/zenith/pull/686/files#diff-2dee9987054422607c1ab7369e102927d8477dea6d75c4c7f3891e3d65b15d1eR254-R260

but to bring that code here, relish_storage interface requires changes which will bloat the PR hence omitted,